Index: users.csv
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>username,password\nyuan,yuan\nhayato,HaKu\ntina,tina\nyuan,yuan\nmiles,miles\nhayato,hayato\njoyce,joyce\njessie,jessie\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/users.csv b/users.csv
--- a/users.csv	(revision 031c383d36a008e5390ab785754c0a14c46caa79)
+++ b/users.csv	(date 1763400515795)
@@ -1,9 +1,7 @@
 username,password
 yuan,yuan
-hayato,HaKu
-tina,tina
-yuan,yuan
-miles,miles
+jessie,jessie
 hayato,hayato
 joyce,joyce
-jessie,jessie
+tina,tina
+miles,miles
