Index: src/main/java/use_case/flight_filter/FlightFilterOutputBoundary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFilterOutputBoundary.java b/src/main/java/use_case/flight_filter/FlightFilterOutputBoundary.java
new file mode 100644
--- /dev/null	(date 1763059828247)
+++ b/src/main/java/use_case/flight_filter/FlightFilterOutputBoundary.java	(date 1763059828247)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public interface FlightFilterOutputBoundary {
+}
Index: src/main/java/use_case/flight_filter/FlightFilterInteractor.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFilterInteractor.java b/src/main/java/use_case/flight_filter/FlightFilterInteractor.java
new file mode 100644
--- /dev/null	(date 1763059780035)
+++ b/src/main/java/use_case/flight_filter/FlightFilterInteractor.java	(date 1763059780035)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public class FlightFilterInteractor {
+}
Index: src/main/java/use_case/flight_filter/FlightFliterInputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFliterInputData.java b/src/main/java/use_case/flight_filter/FlightFliterInputData.java
new file mode 100644
--- /dev/null	(date 1763059703444)
+++ b/src/main/java/use_case/flight_filter/FlightFliterInputData.java	(date 1763059703444)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public class FlightFliterInputData {
+}
Index: src/main/java/use_case/flight_filter/FlightFilterInputBoundary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFilterInputBoundary.java b/src/main/java/use_case/flight_filter/FlightFilterInputBoundary.java
new file mode 100644
--- /dev/null	(date 1763059614655)
+++ b/src/main/java/use_case/flight_filter/FlightFilterInputBoundary.java	(date 1763059614655)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public interface FlightFilterInputBoundary {
+}
Index: src/main/java/interface_adapter/logged_in/FlightFilterController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/interface_adapter/logged_in/FlightFilterController.java b/src/main/java/interface_adapter/logged_in/FlightFilterController.java
new file mode 100644
--- /dev/null	(date 1763061779826)
+++ b/src/main/java/interface_adapter/logged_in/FlightFilterController.java	(date 1763061779826)
@@ -0,0 +1,4 @@
+package interface_adapter.logged_in;
+
+public class FlightFilterController {
+}
Index: src/main/java/data_access/FlightFilterUserDataAccessObject.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/data_access/FlightFilterUserDataAccessObject.java b/src/main/java/data_access/FlightFilterUserDataAccessObject.java
new file mode 100644
--- /dev/null	(date 1763060105313)
+++ b/src/main/java/data_access/FlightFilterUserDataAccessObject.java	(date 1763060105313)
@@ -0,0 +1,4 @@
+package data_access;
+
+public class FlightFilterUserDataAccessObject {
+}
Index: src/main/java/use_case/flight_filter/FlightFilterUserDataAccessInterface.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFilterUserDataAccessInterface.java b/src/main/java/use_case/flight_filter/FlightFilterUserDataAccessInterface.java
new file mode 100644
--- /dev/null	(date 1763059931149)
+++ b/src/main/java/use_case/flight_filter/FlightFilterUserDataAccessInterface.java	(date 1763059931149)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public interface FlightFilterUserDataAccessInterface {
+}
Index: src/main/java/use_case/flight_filter/FlightFilterOutputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/use_case/flight_filter/FlightFilterOutputData.java b/src/main/java/use_case/flight_filter/FlightFilterOutputData.java
new file mode 100644
--- /dev/null	(date 1763059881219)
+++ b/src/main/java/use_case/flight_filter/FlightFilterOutputData.java	(date 1763059881219)
@@ -0,0 +1,4 @@
+package use_case.flight_filter;
+
+public class FlightFilterOutputData {
+}
Index: src/main/java/interface_adapter/logged_in/FlightFilterPresenter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/interface_adapter/logged_in/FlightFilterPresenter.java b/src/main/java/interface_adapter/logged_in/FlightFilterPresenter.java
new file mode 100644
--- /dev/null	(date 1763061779814)
+++ b/src/main/java/interface_adapter/logged_in/FlightFilterPresenter.java	(date 1763061779814)
@@ -0,0 +1,4 @@
+package interface_adapter.logged_in;
+
+public class FlightFilterPresenter {
+}
Index: src/main/java/entity/FlightFilter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/entity/FlightFilter.java b/src/main/java/entity/FlightFilter.java
new file mode 100644
--- /dev/null	(date 1763061248285)
+++ b/src/main/java/entity/FlightFilter.java	(date 1763061248285)
@@ -0,0 +1,4 @@
+package entity;
+
+public class FlightFilter {
+}
Index: src/main/java/view/FlightFilterView.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/view/FlightFilterView.java b/src/main/java/view/FlightFilterView.java
new file mode 100644
--- /dev/null	(date 1763061958718)
+++ b/src/main/java/view/FlightFilterView.java	(date 1763061958718)
@@ -0,0 +1,4 @@
+package view;
+
+public class FlightFilterView {
+}
Index: src/main/java/interface_adapter/logged_in/FlightFilterViewModel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/interface_adapter/logged_in/FlightFilterViewModel.java b/src/main/java/interface_adapter/logged_in/FlightFilterViewModel.java
new file mode 100644
--- /dev/null	(date 1763061909204)
+++ b/src/main/java/interface_adapter/logged_in/FlightFilterViewModel.java	(date 1763061909204)
@@ -0,0 +1,4 @@
+package interface_adapter.logged_in;
+
+public class FlightFilterViewModel {
+}
